sfge/src/main/java/com/salesforce/graph/symbols/ClassStaticScope.java

```diff
@@ -121,9 +160,13 @@ private static List<FieldDeclarationVertex> getFieldDeclarations(
     public static Optional<ApexPath> getInitializationPath(
             GraphTraversalSource g, String classname) {
         ClassStaticScope classStaticScope = ClassStaticScope.get(g, classname);
+		if (classStaticScope.getState().equals(State.INITIALIZED)) {
+			throw new ProgrammingException("Initialization path does not need to be invoked on a class that's already initialized: " + classStaticScope.getClassName());
+		}
         List<BaseSFVertex> vertices = new ArrayList<>();
         vertices.addAll(classStaticScope.getFields());
         vertices.addAll(getFieldDeclarations(g, classStaticScope.userClass));
+		vertices.addAll(getStaticBlocks(g, classStaticScope.userClass));
```

Comment:
Adding calls to static blocks as another item needed to set a class's static scope.